### ISSUE TITLE
Set up metrics for database server

### ISSUE THREAD

**Issue by rossjrw** [croque-scp/notifier#61]
I currently don't have any metrics set up for the database server after #57, beyond what AWS provides by default. The default metrics don't touch the EBS volume - it will at some point run out of storage. I will need to have metrics set up before that happens.

The EBS volume is 8GB. Last I checked, the database was about 300MB? I think? It should be a while before it starts hitting a limit, but I want to be on top of things by the time that happens.

**Comment by rossjrw** [comment]
I believe this is now done. I have set up the CloudWatch agent on the server and configured it to report the default 'Basic' preset of stats, which includes disk usage % and memory usage %. These have been added to the dashboard.

Getting this working was a bit of hassle so I'll document what I remember here for next time.

The database is on a private subnet in a private VPC without internet access. I can temporarily add internet access with an Elastic IP for the purposes of installing the agent, but am unwilling to keep it that access permanently. Instead I needed to create VPC Endpoints in the notifier VPC - 3 in total, all of the form *.eu-west-2.amazonaws.com, for each of `logs`, `monitoring` and `ec2`. I created a self-referencing security group and added to it all three of those and the database instance.

**Comment by rossjrw** [comment]
Turns out that VPC Endpoints are mad expensive. Not prohibitively so, but after all my fine-tuning, it adds a hefty % to the total cost.

I have disabled all of the VPC endpoints and moved the database to a new instance, which is permitted to have an IP assigned to it. There's already internet gateways in the VPC so now no endpoints are needed (and probably weren't in the first place). The database was never in fact on a private subnet at all.
